### DISCUSSION TITLE
Prebuilt Runner Images

### DISCUSSION THREAD

**Discussion by clarkmcc** [actions/virtual-environments#6299] [General questions] [answered]
Are there any pre-built images that can be downloaded so that I don't have to spin up a bunch of Azure infrastructures to generate my own? I've just spent the last hour trying to set up an Azure account with the correct resource groups, client app, and whatever else it needs, and still haven't been able to get image generation to work.

**Comment by Mikebeem** [comment, marked as answer]
I guess not, see this comment in another discussion: [Publishing VM images to public #3919 (comment)](https://github.com/actions/runner-images/discussions/3919#discussioncomment-1198745)
